Source/JavaScriptCore/wasm/WasmSlowPaths.cpp

```diff
@@ -418,14 +418,23 @@ WASM_SLOW_PATH_DECL(array_get)
     if (JSValue::decode(arrayref).isNull())
         WASM_THROW(Wasm::ExceptionType::NullArrayGet);
     uint32_t index = READ(instruction.m_index).unboxedUInt32();
-
     JSValue arrayValue = JSValue::decode(arrayref);
     ASSERT(arrayValue.isObject());
     JSWebAssemblyArray* arrayObject = jsCast<JSWebAssemblyArray*>(arrayValue.getObject());
     if (index >= arrayObject->size())
         WASM_THROW(Wasm::ExceptionType::OutOfBoundsArrayGet);
-
-    WASM_RETURN(Wasm::operationWasmArrayGet(instance, instruction.m_typeIndex, arrayref, index));
+    Wasm::GCOpType arrayGetKind = static_cast<Wasm::GCOpType>(instruction.m_arrayGetKind);
```

Comment:
This used to be split into three slow path functions which called a helper with a callback, but it's now just all in `array_get` as we collapsed the opcodes.